src/app/application-management/application-management.reducer.ts

```diff
@@ -25,6 +26,7 @@ export interface ApplicationManagementState {
         buildPlan?: Plan;
         terminationPlan?: Plan;
         instances: Map<string, ServiceTemplateInstance>;
+        interfaces: Interface[];
```

Comment:
Doesn't this replace buildPlan and terminationPlan?

Reply:
correct, resolved.